src/main/java/duke/Launcher.java

```diff
@@ -0,0 +1,11 @@
+package duke;
+
+import duke.ui.Gui;
+
+import javafx.application.Application;
+
+public class Launcher {
```

Comment:
Think you can add java docs header about what this class does. Eg, Responsible for launching javafx interface